ci/docker/debug-cuda-scalapack.yaml

```diff
@@ -25,6 +25,8 @@ spack:
       variants:
         - 'build_type=Release'
     pika:
+      version:
+        - main
```

Comment:
To do: require 0.19.0 once it's out.

Reply:
Thanks, I did update the requirement in `CMakeLists.txt` and `package.py`, but forgot to remove these :upside_down_face:. These are back to normal now.